test/cpp/test_xla_sharding.cpp

```diff
@@ -64,5 +64,81 @@ TEST_F(XLAShardingTest, ShardTensor) {
   EXPECT_EQ(shards[7].sizes(), c10::ArrayRef<long>({8, 7, 4}));
 }
 
+TEST_F(XLAShardingTest, CreateTensorsData) {
```

Comment:
nit: `TestCreateTensorsData`  instead of `CreateTensorsData`. same for other tests

Reply:
I think {TEST_NAME}.{FUNCTION_IN_TEST}, e.g., `XLAShardingTest.CreateTensorsData` better, as-is?

Reply:
It is to be consistent with existing c++ test https://github.com/pytorch/xla/blob/master/test/cpp/test_aten_xla_tensor.cpp

Reply:
+1 Thanks, consistency is more important